src/main/java/ulcrs/Server.java

```diff
@@ -32,6 +34,7 @@ public static void main(String args[]) {
             path("/schedule", server.scheduleController.routes());
             path("/shift", server.shiftController.routes());
             path("/session", server.sessionController.routes());
+            path("/ulc", server.ulcController.routes());

```

Comment:
In practice, I don't think we'll need this exposed to the frontend. Any calls to the ULC server should be made from the DataStore, so although the UlcController may be required for this purpose, I don't think we want this exposed in our REST API.